src/coffee/sync/utils/base.coffee

```diff
@@ -17,8 +17,9 @@ class BaseUtils
         includeValueOnMove: false  # value of items moved is not included in deltas
       textDiff:
         # if value to diff has a bigger length, a text diffing algorithm is used
+        # disable the text diffing algorithm by passing an unreachable minimum length
         # https://github.com/benjamine/jsondiffpatch/blob/master/docs/deltas.md#text-diffs
-        minLength: 300
+        minLength: Infinity
```

Comment:
C'mon, seriously?!?

I still don't understand and see how and what kind of a problem you are having.
I tried to reproduce this problem (in the new version of the sdk, I haven't tried in the 1.x) but I don't see any.

```js
describe('long text diff patch', () => {
  it('should correctly patch it', () => {
    const before = {
      attributes: [
        {
          name: 'textAttribute',
          value: '[{"textAttributeValue":{"fr-CH":"","de-CH":"","it-CH":"","de-DE":"<p><strong>Some random text to make this longer than the value that was in jsondiffpatch.textDiff.minLength = 300. Also this will be badly formatted JSON”</p>","en-GB":"","es-ES":"","fr-FR":""fr-CH":"","fr-FR": "","it-IT": "","nl-NL": "","ru-RU": ""},"testberichte_video": ""}]',
        },
      ],
    }
    const now = {
      attributes: [
        {
          name: 'textAttribute',
          value: '[{"textAttributeValue":{"fr-CH":"","de-CH":"","it-CH":"","de-DE":"<p><strong>Some random text to make this longer than the value that was in jsondiffpatch.textDiff.minLength = 300. This should be now a correctly formatted JSON. However, after jsondiffpatch, it will be changed into a different string”</p>","en-GB":"","es-ES":"","fr-FR":""}}]'
        },
      ],
    }
    const delta = diffpatcher.diff(before, now)
    expect(JSON.parse(
      diffpatcher.getDeltaValue(
        delta.attributes['0'].value,
        before.attributes[0].value,
      )
    )).toEqual([
      {
        textAttributeValue: {
          'fr-CH': '',
          'de-CH': '',
          'it-CH': '',
          'de-DE': '<p><strong>Some random text to make this longer than the value that was in jsondiffpatch.textDiff.minLength = 300. This should be now a correctly formatted JSON. However, after jsondiffpatch, it will be changed into a different string”</p>',
          'en-GB': '',
          'es-ES': '',
          'fr-FR': '',
        },
      },
    ])
  })
})

// Delta
{
  "attributes": {
    "0": {
      "value": [
        "@@ -178,166 +178,165 @@\n 00. \n-Also this will be badly formatted JSON%E2%80%9D%3C/p%3E%22,%22en-GB%22:%22%22,%22es-ES%22:%22%22,%22fr-FR%22:%22%22fr-CH%22:%22%22,%22fr-FR%22: %22%22,%22it-IT%22: %22%22,%22nl-NL%22: %22%22,%22ru-RU%22: %22%22%7D,%22testberichte_video%22: %22%22\n+This should be now a correctly formatted JSON. However, after jsondiffpatch, it will be changed into a different string%E2%80%9D%3C/p%3E%22,%22en-GB%22:%22%22,%22es-ES%22:%22%22,%22fr-FR%22:%22%22%7D\n %7D%5D\n",
        0,
        2
      ]
    },
    "_t": "a"
  }
}
```